openff/interchange/components/interchange.py

```diff
@@ -425,12 +439,19 @@ def to_top(
         _merge_atom_types: book, default=False
             The flag to define behaviour of GROMACSWriter. If True, then similar atom types will be merged.
             If False, each atom will have its own atom type.
+        _normalize_charges: bool, default = False
+            If True charges written to the topology are normalized per molecule to ensure that each molecule has a
+            charge which is exactly integer. Charges that are exactly 0 are not touched.
+            If False, charges are untouched.
 
         Notes
         -----
         Molecule names in written files are not guaranteed to match the `Moleclue.name` attribute of the
         molecules in the topology, especially if they are empty strings or not unique.
 
+        Charges written to the topology are normalized per molecule to ensure that each molecule has a charge which is
+        exactly integer. Charges that are exactly 0 are not touched.
+
```

Comment:
Since this is not the default behavior and the opt-in would be private here, I think this information should only live in the argument's description